tests/fixtures/get_data_dictionary.py

```diff
@@ -0,0 +1,44 @@
+import os
+import pytest
+
+from phenopy.d2p import load as load_d2p
+from phenopy.network import annotate
+from phenopy.network import load as load_network
+from phenopy.score import Scorer
+from phenopy.util import generate_alternate_ids, read_phenotype_groups
+
+
+@pytest.fixture(scope="function")
```

Comment:
nit - function is the default scope of a fixture, but it never hurts to be explicit.